internal/scorecard/alpha/testpod.go

```diff
@@ -47,12 +47,38 @@ func getPodDefinition(test Test, o Scorecard) *v1.Pod {
 					Image:           "quay.io/operator-framework/scorecard-test:dev",
 					ImagePullPolicy: v1.PullIfNotPresent,
 					Command:         test.Entrypoint,
+					VolumeMounts: []v1.VolumeMount{
+						{
+							MountPath: "/scorecard-bundle-path",
```

Comment:
This is an external interface since custom test images will use this path. Can we use a simpler and easier to remember location?
```suggestion
							MountPath: "/bundle",
```

Reply:
renamed based on your suggestion.